packages/mui-styles/src/makeStyles/makeStyles.test.js

```diff
@@ -44,72 +23,73 @@ describe('makeStyles', () => {
 
   it('should accept a classes prop', () => {
     const styles = { root: {} };
-    const mountWithProps = createGetClasses(styles);
-    const output = mountWithProps();
-    const baseClasses = output.classes;
-    output.wrapper.setProps({
-      classes: { root: 'h1' },
-    });
-    const extendedClasses = output.classes;
+    const useStyles = makeStyles(styles);
+
+    const { result, rerender } = renderHook((props) => useStyles(props));
```

Comment:
Using `renderHook` simplifies the tests setup and probably makes them easier to read by removing abstractions.